### DISCUSSION TITLE
Install Netclient on Edgerouter (ER6P

### DISCUSSION THREAD

**Discussion by DingoBlue** [gravitl/netmaker#1450] [Q&A]
Has anyone made any progress or had any succcess getting Netclient running on an Edgerouter?

I would like to use 2 remote Edgerouters (ER6P) as egress/ingress gateways but am having difficulty.
I have 2 x ER6P's (1 remote, 1 local) running latest Edgeos (v2.0.9-hotfix.4).  I have Wireguard installed on the ER6P and have come across a Wireguard wizard for Edgerouter https://github.com/vchrizz/ER-wizard-WireGuard which I hoped would give me a few clues but I am still "clueless".  I have tried the different Netclient distros but none are compatable with the latest MIPS os (some kind of Debian flavour I think?) so cannot load Netclient.
Regards

**Comment by afeiszli** [comment]
We have no experience running on edgerouter, but if you're just looking to build a compatible netclient binary, you can run a "go build" with the parameters necessary for the OS/arch. See some examples here: https://github.com/gravitl/netmaker/blob/master/.github/workflows/buildandrelease.yml#L197
